pkg/gadgets/trace/capabilities/types/types.go

```diff
@@ -35,15 +36,19 @@ type Event struct {
 	eventtypes.Event
 	eventtypes.WithMountNsID
 
-	Pid     uint32 `json:"pid,omitempty" column:"pid,template:pid"`
-	Comm    string `json:"comm,omitempty" column:"comm,template:comm"`
-	Syscall string `json:"syscall,omitempty" column:"syscall,template:syscall"`
-	UID     uint32 `json:"uid,omitempty" column:"uid,minWidth:6"`
-	Cap     int    `json:"cap,omitempty" column:"cap,width:3,fixed"`
-	CapName string `json:"capName,omitempty" column:"capName,width:18,fixed"`
-	Audit   int    `json:"audit,omitempty" column:"audit,minWidth:5"`
-	Verdict string `json:"verdict,omitempty" column:"verdict,width:7,fixed"`
-	InsetID *bool  `json:"insetid,omitempty" column:"insetid,width:7,fixed,hide"`
+	Pid           uint32   `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm          string   `json:"comm,omitempty" column:"comm,template:comm"`
+	Syscall       string   `json:"syscall,omitempty" column:"syscall,template:syscall"`
+	UID           uint32   `json:"uid,omitempty" column:"uid,minWidth:6"`
+	Cap           int      `json:"cap,omitempty" column:"cap,width:3,fixed"`
+	CapName       string   `json:"capName,omitempty" column:"capName,width:18,fixed"`
+	Audit         int      `json:"audit,omitempty" column:"audit,minWidth:5"`
+	Verdict       string   `json:"verdict,omitempty" column:"verdict,width:7,fixed"`
+	InsetID       *bool    `json:"insetid,omitempty" column:"insetid,width:7,fixed,hide"`
+	TargetUserNs  uint64   `json:"targetuserns,omitempty" column:"targetuserns,template:ns"`
+	CurrentUserNs uint64   `json:"currentuserns,omitempty" column:"currentuserns,template:ns"`
+	Caps          uint64   `json:"caps,omitempty" column:"caps,hide"`
+	CapsNames     []string `json:"capsNames,omitempty" column:"capsnames,hide"`
```

Comment:
@flyth FYI, I named the column `targetuserns`, `currentuserns`, `capsnames` in lower case because it was done like this for some columns like `cumulruncount`. But there are other columns with different casing. Do you have a recommended casing?

Reply:
The columns library itself is case insensitive and the headers are always uppercase when displayed right now - so the exact name can only be seen in the defaults / available columns output (or: it doesn't really matter right now). But we should definitely agree on a standard format and keep it consistent across all gadgets. I personally like camel case more than having all lower caps.